# Translation of OpenERP Server.
# This file contains the translation of the following modules:
# * auth_oauth
# 
# Translators:
# FIRST AUTHOR <EMAIL@ADDRESS>, 2013
msgid ""
msgstr ""
"Project-Id-Version: Odoo 7.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-08-14 00:10+0000\n"
"PO-Revision-Date: 2015-07-17 08:37+0000\n"
"Last-Translator: Martin Trigaux\n"
"Language-Team: Hungarian (http://www.transifex.com/odoo/odoo-7/language/hu/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: auth_oauth
#: field:auth.oauth.provider,validation_endpoint:0
msgid "Validation URL"
msgstr "URL elérési út érvényesítés"

#. module: auth_oauth
#: field:auth.oauth.provider,auth_endpoint:0
msgid "Authentication URL"
msgstr "URL elérési út hitelesítés"

#. module: auth_oauth
#: model:ir.model,name:auth_oauth.model_base_config_settings
msgid "base.config.settings"
msgstr "base.config.settings"

#. module: auth_oauth
#. openerp-web
#: code:addons/auth_oauth/static/src/js/auth_oauth.js:14
#, python-format
msgid "Sign up error"
msgstr ""

#. module: auth_oauth
#: field:auth.oauth.provider,name:0
msgid "Provider name"
msgstr "Szolgáltató neve"

#. module: auth_oauth
#: field:auth.oauth.provider,scope:0
msgid "Scope"
msgstr "Hatáskör"

#. module: auth_oauth
#: field:res.users,oauth_provider_id:0
msgid "OAuth Provider"
msgstr "OAuth szolgáltató"

#. module: auth_oauth
#: field:auth.oauth.provider,css_class:0
msgid "CSS class"
msgstr "CSS osztály"

#. module: auth_oauth
#: field:auth.oauth.provider,body:0
msgid "Body"
msgstr "Törzs"

#. module: auth_oauth
#: model:ir.model,name:auth_oauth.model_res_users
msgid "Users"
msgstr "Felhasználók"

#. module: auth_oauth
#: field:auth.oauth.provider,sequence:0
msgid "unknown"
msgstr "ismeretlen"

#. module: auth_oauth
#. openerp-web
#: code:addons/auth_oauth/static/src/js/auth_oauth.js:16
#: code:addons/auth_oauth/static/src/js/auth_oauth.js:18
#, python-format
msgid "Authentication error"
msgstr ""

#. module: auth_oauth
#: field:res.users,oauth_access_token:0
msgid "OAuth Access Token"
msgstr "OAuth hozzáférési Token"

#. module: auth_oauth
#: field:auth.oauth.provider,client_id:0
#: field:base.config.settings,auth_oauth_facebook_client_id:0
#: field:base.config.settings,auth_oauth_google_client_id:0
msgid "Client ID"
msgstr "Ügyfál ID azonosító"

#. module: auth_oauth
#. openerp-web
#: code:addons/auth_oauth/static/src/js/auth_oauth.js:16
#, python-format
msgid "Access Denied"
msgstr "A hozzáférés megtagadva"

#. module: auth_oauth
#: model:ir.ui.menu,name:auth_oauth.menu_oauth_providers
msgid "OAuth Providers"
msgstr "OAuth szolgáltatók"

#. module: auth_oauth
#: model:ir.model,name:auth_oauth.model_auth_oauth_provider
msgid "OAuth2 provider"
msgstr "OAuth2 szolgáltató"

#. module: auth_oauth
#: field:res.users,oauth_uid:0
msgid "OAuth User ID"
msgstr "OAuth felhasználó ID azonosító"

#. module: auth_oauth
#: field:base.config.settings,auth_oauth_facebook_enabled:0
msgid "Allow users to sign in with Facebook"
msgstr "Felhasználók Facebook keresztüli bejelentkezés engedélyezése"

#. module: auth_oauth
#: sql_constraint:res.users:0
msgid "OAuth UID must be unique per provider"
msgstr "OAuth UID egyedinek kell lennie mindegyik szolgáltatóhoz"

#. module: auth_oauth
#: help:res.users,oauth_uid:0
msgid "Oauth Provider user_id"
msgstr "Oauth szolgáltatói felhasználó_id"

#. module: auth_oauth
#: field:auth.oauth.provider,data_endpoint:0
msgid "Data URL"
msgstr "Adat  URL elérési út"

#. module: auth_oauth
#: view:auth.oauth.provider:0
msgid "arch"
msgstr "vezető"

#. module: auth_oauth
#: model:ir.actions.act_window,name:auth_oauth.action_oauth_provider
msgid "Providers"
msgstr "Szolgáltatók"

#. module: auth_oauth
#: field:base.config.settings,auth_oauth_google_enabled:0
msgid "Allow users to sign in with Google"
msgstr "Engedélyezze a felhasználók Google bejelentkezését"

#. module: auth_oauth
#: field:auth.oauth.provider,enabled:0
msgid "Allowed"
msgstr "Engedélyezett"

#. module: auth_oauth
#. openerp-web
#: code:addons/auth_oauth/static/src/js/auth_oauth.js:14
#, python-format
msgid "Sign up is not allowed on this database."
msgstr "A feliratkozás nem megengedett erre az adatbázisra."

#. module: auth_oauth
#. openerp-web
#: code:addons/auth_oauth/static/src/js/auth_oauth.js:18
#, python-format
msgid ""
"You do not have access to this database or your invitation has expired. "
"Please ask for an invitation and be sure to follow the link in your "
"invitation email."
msgstr "Nincs elérési jogosultsága ehhez az adatbázishoz vagy a meghívója lejárt. Kérjen meghívót és győződjön meg róla, hogy az e-mail-ben elküldött hivatkozást használja."
